src/settings.cpp

```diff
@@ -147,6 +153,10 @@ void Settings::load()
 
     mNotificationDefaultColor = QColor( mSettings->value(
             "common/defaultcolor", mNotificationDefaultColor.name() ).toString() );
+    mNotificationBorderColor = QColor(mSettings->value(
+            BORDER_COLOR_KEY, mNotificationBorderColor.name()).toString());
+    mNotificationBorderWidth = mSettings->value(
+            BORDER_WIDTH_KEY, mNotificationBorderWidth).toUInt();
```

Comment:
A check like this:
```suggestion
            BORDER_WIDTH_KEY, mSettings->value("common/defaultcolor").isNull() ? 0 : mNotificationBorderWidth).toUInt();
```

Reply:
Not on existing installations; for new its fine.